src/main/java/cms/gov/madie/measure/utils/TestCaseServiceUtil.java

```diff
@@ -348,6 +351,66 @@ && isEmpty(group.getStratificationValues()))
     return new ArrayList<>(populationCriteria);
   }
 
+  public static List<TestCaseGroupPopulation> assignStratificationValuesQiCore(
+      List<TestCaseGroupPopulation> testCaseGroupPopulations, List<Group> measureGroups) {
+
+    // Break up single list of pop values and strats into separate lists
+    List<TestCaseGroupPopulation> populationCriteria =
+        testCaseGroupPopulations.stream()
+            .filter(
+                group -> {
+                  return isNotEmpty(group.getPopulationValues());
+                  // && isEmpty(group.getStratificationValues());
+                })
+            .toList();
+
+    List<TestCaseStratificationValue> stratification =
+        testCaseGroupPopulations.stream()
+            .filter(
+                group -> {
+                  return isNotEmpty(group.getStratificationValues());
+                })
+            // Assumes there cannot be more than 1 strat in each incoming expected value obj
+            // .map(group -> group.getStratificationValues().get(0))
+            // (GAK MAT-8064 Why???  The QICore Test case can have a single population with multiple
+            // stratification results)
+            .flatMap(group -> group.getStratificationValues().stream())
+            .toList();
+
+    // Mismatch between target and import Stratification, don't set any expected values
+    boolean measureHasStrats =
+        measureGroups != null
+            ? measureGroups.stream().allMatch(group -> isNotEmpty(group.getStratifications()))
+            : false;
```

Comment:
I think this is works as a simplification.
```suggestion
measureGroups != null && measureGroups.stream().anyMatch(group -> isEmpty(group.getStratifications()));
```

Reply:
That would assign true if "anyMatch" instead of only true if "allMatch", right.  Or am I misreading?